content/doc/book/security/securing-jenkins.adoc

```diff
@@ -60,7 +60,7 @@ We recommend you read them first and act on them immediately.
 
 * https://wiki.jenkins.io/display/JENKINS/Security+implication+of+building+on+master[Security implication of building on controller] --- protect Jenkins controller from malicious builds
 * https://wiki.jenkins.io/display/JENKINS/Securing+JENKINS_HOME[Securing JENKINS_HOME] --- protect Jenkins from users with local access
-* link:environment-variables[Handling Environment Variables] -- ensure that environment variables passed to build steps do not have unintended side effects
+* https://www.jenkins.io/doc/book/security/environment-variables[Handling Environment Variables] -- ensure that environment variables passed to build steps do not have unintended side effects
```

Comment:
We generally prefer to keep the URL without the site so that we can run the site on a development machine and navigate links without navigating away from the development machine.
```suggestion
* link:/doc/book/security/environment-variables[Handling Environment Variables] -- ensure that environment variables passed to build steps do not have unintended side effects
```